ontology/yaml/resources/PLUMBING/entity_types/RO.yaml

```diff
@@ -28,4 +28,7 @@ RO_STD:
   - control_mode
   - line_pressure_sensor
   - water_totaldissolvedsolids_concentration_sensor
+  opt_uses:
+  - high_pressure_status
```

Comment:
Can you please provide more detail about the intended use of this field?

Reply:
This pressure status are used to turn On & Off the transfer pumps associated with the RO.

Reply:
Closing this pull request as low_pressure_status , high_pressure_status points are not been taken on cloud.